apps/website-25/src/pages/_app.tsx

```diff
@@ -2,15 +2,50 @@ import '../globals.css';
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
 import dynamic from 'next/dynamic';
+import posthog from 'posthog-js';
+/* eslint-disable import/no-extraneous-dependencies */
+import { PostHogProvider } from 'posthog-js/react';
 import {
   CookieBanner, Footer, isCurrentPath, Nav, constants,
 } from '@bluedot/ui';
 import clsx from 'clsx';
+import { useEffect } from 'react';
+import { Router } from 'next/router';
 import { Analytics } from '../components/Analytics';
 
+if (typeof window !== 'undefined') {
+  if (!process.env.NEXT_PUBLIC_POSTHOG_KEY) {
+    console.error('PostHog key is missing! Check your .env.local file');
+  } else {
+    console.log('Initializing PostHog...');
+    posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+      api_host: 'https://eu.i.posthog.com',
+      person_profiles: 'always',
+      // TODO: Do we want this to be "identified_only" or "always"?
```

Comment:
cc @domdomegg

Reply:
I think we want this to be `identified_only` or just unset. We don't currently link PostHog data to individuals, and haven't properly considered the privacy implications yet.

(I think in practice though `always` is fine for now given we don't actually identify people anywhere, so it wastes a bit of data but otherwise does not have privacy implications)

Reply:
Updated the comment to reflect this